prober/prober.dockerfile

```
@@ -0,0 +1,15 @@
+# Base image that supports bash
+FROM cgr.dev/chainguard/bash:latest
+
+COPY jvsctl /jvsctl
+COPY ./prober/prober.sh prober.sh
```

nit: to be consistent
```suggestion
COPY ./prober/prober.sh /prober.sh
```

---

Done.